packages/paste-libraries/clipboard-copy/package.json

```diff
@@ -0,0 +1,39 @@
+
+{
+  "name": "@twilio-paste/clipboard-copy-library",
+  "version": "0.2.1",
```

Comment:
nit: Probably fine to start this as a 0.0.0 as it's the initial commit. I know the release will do a major bump, but kinda seems weird to start the commit history at 0.2.1

Reply:
does it also update the bump for twilio-website here, or will it say `0.0.0`? https://github.com/twilio-labs/paste/pull/1799/files#diff-b0ac619a51509a0f1df92fddafe2a0cee283024c61c9d185d3cf5dbd97218582R35

Reply:
Yeah, Changesets deals with bumping all the peer dep versions across the monorepo. It's got your back

Reply:
sounds good, updated 👍